GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -435,6 +435,38 @@ protected override void OnHandleCreated(EventArgs e)
             base.OnHandleCreated(e);
         }
 
+        public void FixForOwnerScreenLocation(IWin32Window? owner)
+        {
+            if (owner != null)
```

Comment:
Our coding style prefers Early Exit, i.e.
```suggestion
            if (owner == null)
            {
                return;
            }
```

Reply:
Null check should be done with `is null` and `is not null`.